cfg/rippled-example.cfg

```diff
@@ -1359,7 +1365,14 @@ advisory_delete=0
 # https://xrpl.org/history-sharding.html
 #[shard_db]
 #path=/var/lib/rippled/db/shards/nudb
-#max_size_gb=500
```

Comment:
Was this setting removed or just renamed?

Reply:
I'd say it was removed since `max_historical_shards` has a different meaning.

Reply:
Then having "500" there is going to be unexpectedly large for a lot of people.

Also `max_historical_shards` isn't documented around line 960, only here.

Lastly having a section in the "actual" configuration section yet completely commenting it out seems like a strange choice. If you're only concerned about giving an example on how to configure the `[shard_db]` section, you already have the (nearly) full documentation in line ~950. If this is really that important for the network health, how about uncommenting this section and storing let's say a dozen historic shards by default?

Reply:
Set the commented value for `max_historical_shards` to 50 and added documentation for this optional field. There's already a precedent for providing commented-out sections as starter templates so I'm inclined to leave that as is.